libs/core/src/lib/utils/abstract-fd-ngx-class.ts

```diff
@@ -21,6 +21,14 @@ export abstract class AbstractFdNgxClass implements OnInit, OnChanges {
     /** @hidden */
     abstract _setProperties(): void;
 
+    _setClassToElement(className: string) {
```

Comment:
Return type and shouldn't it be named `setClassToElement` (no prefix)?

Reply:
true although other functions have the same prefix so in order to not break anything I left code style here untouched

Reply:
return type added